src/Middleware/src/Headstart.API/Commands/HSProductCommand.cs

```diff
@@ -531,7 +534,7 @@ public async Task Delete(string id, string token)
 			{
 				tasks.Add(Throttler.RunAsync(product.xp.Images, 100, 5, i => _assetClient.DeleteAssetByUrl(i.Url)));
 			}
-			if (product?.xp?.Documents.Count() > 0)
+			if (product?.xp?.Documents != null && product?.xp?.Documents.Count() > 0)
```

Comment:
I think now that we aren't using our homegrown cms we can actually just remove this piece of code that tries to delete documents and images associated with the product as those are now simply xp properties on the product.

@kingeric027  can you please confirm?

Reply:
Well if we delete a product wouldn't it also make sense to delete those assets in our blob container? That's what this is doing here.

Reply:
Oh true, nevermind!